### PR TITLE
added endpoint to get all devices

### PR DESCRIPTION
**Pull request by mtran-12**
Pulls all devices from the db, use the endpoint http://localhost:3000/api/getAllDevices to test that it pulls all devices

### CODE REVIEW

**Comment by McGarvs** [comment]
Add a comment for the other endpoint so I can check it.

**Comment by mtran-12** [comment]
No need, already reviewed
it was merged into this branch

File: cradlepoint/pages/api/getAllDevices.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { ObjectId } = require('mongodb');
```
Comment (McGarvs) [inline cradlepoint/pages/api/getAllDevices.js:1]: Remove this line

**Review by McGarvs — COMMENTED** [review]
Looks good, just remove that line I'd say

**Pull request merged**